datasets/npr/gebiedsRegelingen/v0.0.1.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "gebiedsRegelingen",
+  "type": "table",
+  "version": "0.0.1",
+  "provenance": "Extractie van RDW, van alle condities die gelden wanneer iemand een recht voor een bepaald gebied verwerft: 'https://opendata.rdw.nl/resource/yefi-qfiq.json'",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "identifier": "regelingId",
+    "required": [
+      "schema",
+      "regelingId"
+    ],
+    "display": "regelingId",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "gebiedsbeheerderId": {
+        "type": "integer",
+        "provenance": "areamanagerid",
+        "description": "Identificatiecode van de gebiedsbeheerder of parkeerexploitant (typisch 363 voor Amsterdam en 457 voor Weesp)."
+      },
+      "regelingId": {
+        "type": "string",
+        "provenance": "regulationid",
+        "description": "Identificatiecode van het gebied waarbinnen een bepaalde regeling geldt."
+      },
+      "regelingOmschrijving": {
+        "type": "string",
+        "provenance": "regulationdesc",
+        "description": "Omschrijving van de regeling."
+      },
+      "beginDatumGebiedsbeheerderId": {
+        "type": "integer",
```

Comment:
The `id` postifx indeed suggest a type of `integer` but the rest of the name and also the description suggests this is a date field.

Reply:
Changed name and format